.github/workflows/ci.yml

```diff
@@ -14,81 +14,185 @@ permissions:
   contents: write
 
 jobs:
+  # check if formatting is correct
   fmt:
-    runs-on: macos-14
+    runs-on: ubuntu-latest
     steps:
       - name: checkout
         uses: actions/checkout@v4
+
       - name: setup rust
-        run: |
-          brew unlink rustup
-          curl --proto '=https' --tlsv1.2 --retry 10 --retry-connrefused -fsSL "https://sh.rustup.rs" | sh -s -- --default-toolchain none -y
-      - name: setup rust toolchain
-        run: |
-          . "$HOME/.cargo/env"
-          which rustup
-          rustup --version
-          rustup toolchain install
+        uses: dtolnay/rust-toolchain@master
+        with:
+          toolchain: 1.91.0
+
       - name: setup verusfmt
-        run: |
-          curl --proto '=https' --tlsv1.2 -LsSf https://github.com/verus-lang/verusfmt/releases/latest/download/verusfmt-installer.sh | sh
+        run: curl --proto '=https' --tlsv1.2 -LsSf https://github.com/verus-lang/verusfmt/releases/latest/download/verusfmt-installer.sh | sh
+
       - name: check rustfmt/verusfmt
         working-directory: ./source
         run: |
-          . "$HOME/.cargo/env"
           . ../tools/activate
           vargo fmt -- --check
+
       - name: check cargo fmt for vargo
         working-directory: ./tools/vargo
+        run: cargo fmt -- --check
+
+  # check if it builds and runs on our main supported platforms
```

Comment:
This comment seems out of date now, right?  This is only running the tests for one platform (macos)